web_src/js/features/repo-issue.ts

```diff
@@ -184,23 +183,26 @@ export function initRepoIssueCommentDelete() {
 
 export function initRepoIssueCodeCommentCancel() {
   // Cancel inline code comment
-  document.addEventListener('click', (e: DOMEvent<MouseEvent>) => {
-    if (!e.target.matches('.cancel-code-comment')) return;
+  document.addEventListener('click', (e) => {
+    if (!(e.target as HTMLElement).matches('.cancel-code-comment')) return;
 
-    const form = e.target.closest('form');
+    const form = (e.target as HTMLElement).closest('form')!;
     if (form?.classList.contains('comment-form')) {
       hideElem(form);
-      showElem(form.closest('.comment-code-cloud')?.querySelectorAll('button.comment-form-reply'));
+      const replyBtn = form.closest('.comment-code-cloud')?.querySelectorAll('button.comment-form-reply');
+      if (replyBtn) {
+        showElem(replyBtn);
+      }
```

Comment:
```suggestion
      showElem(form.closest('.comment-code-cloud')!.querySelectorAll('button.comment-form-reply'));
```

`showElem` doesn't accept `null`, so it's safe to use `!`